classes/class_control.rst

```diff
@@ -566,7 +566,7 @@ Constants
 Description
 -----------
 
-Base class for all User Interface or *UI* related nodes. ``Control`` features a bounding rectangle that defines its extents, an anchor position relative to its parent and margins that represent an offset to the anchor. The margins update automatically when the node, any of its parents, or the screen size change.
+Base class for all User Interface or *UI* related nodes. ``Control`` features a bounding rectangle that defines its extents, an anchor position relative to its parent control or the curent viewport and margins that represent an offset to the anchor. The margins update automatically when the node, any of its parents, or the screen size change.
```

Comment:
Changes to the class reference should be done in the engine repo at https://github.com/godotengine/godot/blob/master/doc/classes/Control.xml

This file is automatically generated, as mentioned in its header:
```
.. Generated automatically by doc/tools/makerst.py in Godot's source tree.
.. DO NOT EDIT THIS FILE, but the Control.xml source instead.
.. The source is found in doc/classes or modules/<name>/doc_classes.
```

Reply:
Also two comments on the style: "current" takes two R-s, and I would add a comma after "viewport".

Reply:
Ok well I did not read the header.  Partly because I assumed that the link on the documentation "Edit on GitHub" would like to where it needs to be edited : ).

Reply:
Yeah sadly the "Edit on GitHub" is inaccurate for the API documentation pages, but can't really be customized :/